tests/unit/Rules/FibonacciTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+/**
+ * @group  rule
+ * @covers Respect\Validation\Rules\Fibonacci
+ * @covers Respect\Validation\Exceptions\FibonacciException
+ */
+class FibonacciTest extends \PHPUnit_Framework_TestCase
```

Comment:
This test case can be a lot simpler, take a look on [ArrayTypeTest](https://github.com/Respect/Validation/blob/c2eece6/tests/unit/Rules/ArrayTypeTest.php).